MaxMind.MinFraud/Exception/HttpException.cs

```diff
@@ -13,6 +14,7 @@ namespace MaxMind.MinFraud.Exception
     ///     by the web service itself. As such, it is a IOException instead of a
     ///     MinFraudException.
     /// </summary>
+    [Serializable]
```

Comment:
The changes to the exceptions are unrelated to the JSON changes. I just made the same changes that I did in the GeoIP2 library, following the recommended pattern.